### PR TITLE
added new food spawn rules

### PR DESCRIPTION
**Pull request by codeallthethingz**

### CODE REVIEW

**Comment by bvanvugt** [comment]
Yeah I've been thinking about how best to describe the new algorithm. I think the simplest terms are something like:

There's a calculated chance of new food spawning on any turn during the game. This chance increases each turn until a new piece of food spawns, at which point it is reset. At most one piece of food will spawn on any turn. How fast food spawns can be configured when creating a game. All tournament games will use the same value for food spawn; this value will be announced during the welcome (??)

**Comment by dlsteuer** [comment]
Yeah that explanation is better them mine.

**Comment by codeallthethingz** [comment]
Does the dev mode in engine reflect this new algorithm as once the docs are correct I'll send a message to the battlesnake slack telling people to update.

File: docs/2019-tournament-game-rules.md [hunk 1]
```diff
@@ -34,13 +34,11 @@ only positions 1 and 2 will be filled.
 
 ## Food Spawn Rules
 
-- The amount of food on a board will be set at the beginning of a game.  
-- You will not know how much food will be on the board before the game, however it is guaranteed that the amount of food will be at least 1.
-- The same amount of food will remain on the board at all times.  That is, if a piece of food is eaten by a snake, a 
-new piece of food will be available on the board in the next turn.
-- Food will spawn at random coordinates*
+- There's a calculated chance of new food spawning on any turn during the game.  This chance increases each turn until a new piece of food spawns, at which point it is reset. At most one piece of food will spawn on any turn.  How fast food spawns can be configured when creating a game.  All tournament games will use the same value for food spawn and this value will be announced on the day of the tournament. 
+- You will not know how much food will be on the board before the game, however it is guaranteed that the starting amount of food will be at least 1.
```
Comment (codeallthethingz) [inline docs/2019-tournament-game-rules.md]: @dlsteuer @bvanvugt  is this line true or should it be removed?

**Pull request merged**